app/src/main/java/io/intrepid/contest/screens/contestcreation/namecontest/NameContestPresenter.java

```diff
@@ -0,0 +1,14 @@
+package io.intrepid.contest.screens.contestcreation.namecontest;
+
+import io.intrepid.contest.base.BaseContract;
+
+public class NameContestPresenter {
+
+    public interface View extends BaseContract.View{
+        void acceptContestName(String contestName);
+    }
+
+    public interface Presenter {
```

Comment:
The presenter has a Presenter interface? Maybe you could rename this to NameContestContract, which is empty?
(But again, you said this stuff will probably change)